third_party/llvm-project/libcxx/src/algorithm.cpp

```diff
@@ -11,9 +11,24 @@
 
 _LIBCPP_BEGIN_NAMESPACE_STD
 
+// The definition for std::__bit_log2 is in 
```

Comment:
@kaidokert  ptal

Reply:
Why does this conflict happen though ? Why is a call site invoking a call to std::__bit_log2 if it's being compiled with c++17 settings ?